.github/scripts/trymerge.py

```diff
@@ -805,13 +808,14 @@ def find_matching_merge_rule(pr: GitHubPR,
                                  f"{', '.join(list(rule_approvers_set)[:5])}{', ...' if len(rule_approvers_set) > 5 else ''}")
             continue
         if rule.mandatory_checks_name is not None:
+            not_run_checks: List[Tuple[str, Optional[str]]] = []
             pending_checks: List[Tuple[str, Optional[str]]] = []
             failed_checks: List[Tuple[str, Optional[str]]] = []
             checks = pr.get_checkrun_conclusions()
             # HACK: We don't want to skip CLA check, even when forced
             for checkname in filter(lambda x: force is False or "CLA Check" in x, rule.mandatory_checks_name):
                 if checkname not in checks:
-                    pending_checks.append((checkname, None))
+                    not_run_checks.append((checkname, None))
```

Comment:
ah so this assumes that the graphql api will not include checks that aren't run but will mark checks with conclusion None if they're pending?